src/configuration.st

```diff
@@ -2,12 +2,23 @@ CONFIGURATION PLC_1
     VAR_GLOBAL
         GlobalInt : int;
```

Comment:
please all keywords in capital letters 

Int, int --> INT
Global variables should start with capital letters. Please refer ton the style guide at our AX documentation